sdcm/cluster.py

```diff
@@ -3108,7 +3108,7 @@ def reload_config(self):
     def get_token_ring_members(self) -> list[dict[str, str]]:
         self.log.debug("Get token ring members")
         token_ring_members = []
-        token_ring_members_cmd = build_node_api_command('/storage_service/host_id')
+        token_ring_members_cmd = build_node_api_command('/storage_service/host_id', silent=False)
```

Comment:
@fgelcer this seems backwards then what you are trying to fix, this command needs to silent

Anyhow I would prefer adding a parameter of extra arguments, and start mapping all of curl command options

Reply:
did i disable silence on the wrong command? i will fix it, but the whole mapping of the CURL command, should be done incrementally IMHO, as i don't think we will use them all

Reply:
@fruch , returned this one to be silent, and remove what is not supposed to be silent